tools/run_tests/run_tests.py

```diff
@@ -325,13 +325,19 @@ def test_specs(self):
         if cpu_cost == 'capacity':
           cpu_cost = multiprocessing.cpu_count()
         if os.path.isfile(binary):
+          test_prefix = None
           if 'gtest' in target and target['gtest']:
-            # here we parse the output of --gtest_list_tests to build up a
-            # complete list of the tests contained in a binary
-            # for each test, we then add a job to run, filtering for just that
-            # test
+            test_prefix = 'gtest'
+          elif 'benchmark' in target and target['benchmark']:
+            test_prefix = 'benchmark'
+
+          if test_prefix:
+            # here we parse the output of --gtest_list_tests (or 
+            # --benchmark_list_tests)to build up a complete list of 
+            # the tests contained in a binary for each test, we then 
+            # add a job to run, filtering for just that test.
             with open(os.devnull, 'w') as fnull:
-              tests = subprocess.check_output([binary, '--gtest_list_tests'],
+              tests = subprocess.check_output([binary, '--%s_list_tests' % test_prefix],
```

Comment:
Cute... but can we explicitly callout gtest_list_tests, benchmark_list_tests?

It'll be a few extra lines, but worth it for slightly less magic (and magic is a penalty in this code)